deploy/examples/operator-openshift.yaml

```diff
@@ -403,6 +403,7 @@ data:
   # Configure CSI RBD grpc and liveness metrics port
   # CSI_RBD_GRPC_METRICS_PORT: "9090"
   # CSI_RBD_LIVENESS_METRICS_PORT: "9080"
+  # CSI_ADDONS_METRICS_PORT: "4080"
```

Comment:
I do not think the sidecar exposes metrics at the moment.

The sidecar does listen on a port, but that is only done so that controller can connect to it, not for metrics